README.md

```diff
@@ -10,19 +10,6 @@ _real_ browsers.
 >  fast, and fun.
 
 
-## Help and Support
-
-> For questions and support please use the mailing list or Gitter.
-> The issue tracker is for bug reports and feature discussions only.
-
-* Obligatory [documentation]
-* Quick questions:
-[![Gitter Chat](https://img.shields.io/badge/GITTER-join%20chat-green.svg?style=flat-square)](https://gitter.im/karma-runner/karma)
-* Longer questions: [Mailing List]
-* Bug reports [Issue Tracker]
-* Everything less than 140 characters: [@JsKarma] on Twitter
```

Comment:
What the point of all this README edit? The original conversational tone was better IMHO.